app/pb_kits/playbook/tokens/_colors.scss

```diff
@@ -1,10 +1,12 @@
+@import "./opacity";
+
 //=====================================
 // Base colors should not be documented.
 // Only document color use.
 
 // Colors -----------------------------
-$royal:               #004DFF;
-$blue:                #0082FF;
+$royal:               #0056CF;
+$blue:                #0056CF;
```

Comment:
Should we remove this one?

Reply:
@jasoncypret  Good catch! I removed the old dark blue and all references to it.

2 replacements had to be made:

### Product Color for Doors
![](https://d2y84jyh761mlc.cloudfront.net/items/2b2F173h2u0A1s2U1n1w/Image%202019-11-14%20at%2011.54.09%20AM.png?X-CloudApp-Visitor-Id=3221729)

### Secondary Color
![](https://d2y84jyh761mlc.cloudfront.net/items/061E0u1A19053l2I3z2D/Image%202019-11-14%20at%2011.56.34%20AM.png?X-CloudApp-Visitor-Id=3221729)